# Generated by Django 3.1.1 on 2020-12-20 11:00

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('api_searcher', '0003_auto_20201218_2305'),
    ]

    operations = [
        migrations.CreateModel(
            name='CausedByModel',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('field', models.CharField(max_length=350)),
                ('process', models.TextField(max_length=350)),
                ('description', models.TextField()),
            ],
        ),
        migrations.CreateModel(
            name='TechnicalImpactModel',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('title', models.TextField()),
            ],
        ),
        migrations.DeleteModel(
            name='CWEDate',
        ),
        migrations.AddField(
            model_name='cwemodel',
            name='description',
            field=models.TextField(default=''),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='cwemodel',
            name='likehood',
            field=models.TextField(default='', max_length=100),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='technicalimpactmodel',
            name='cwe',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='cwe_technical_model', to='api_searcher.cwemodel'),
        ),
        migrations.AddField(
            model_name='causedbymodel',
            name='cwe',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='cwe_caused_by', to='api_searcher.cwemodel'),
        ),
    ]
